src/components/HeaderButtons/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const HeaderButtons = () => {
```

Comment:
you should name it like login. because later there can be many others buttons inside headers